doc/command_ref.rst

```diff
@@ -335,6 +335,14 @@ Options
     Configure DNF as if the distribution release was ``<release>``. This can
     affect cache paths, values in configuration files and mirrorlist URLs.
 
+``--releasever_major=<major version>``
+    Override the releasever_major variable, which is usually automatically
+    detected or taken from the part of ``$releasever`` before the first ``.``.
+
+``--releasever_minor=<minor version>``
+    Override the releasever_minor variable, which is usually automatically
+    detected or taken from the part of ``$releasever`` after the first ``.``.
+
```

Comment:
If I understand correctly and these new options don't affect the `releasever` variable I would prefer to be explicit about it here as well.
Personally I would be surprised by this.

Reply:
Done.